src/Main.cc

```diff
@@ -229,6 +229,31 @@ static void handler(int sig)
   raise(sig);
 }
 
+namespace {
+// Done launching, show splash and switch to working folder
+bool launched(int& argc, char* argv[])
+{
+  bool no_splash = findCmdLineFlag(argc, argv, "-no_splash");
+  if (!no_splash) {
+    showSplash();
+  }
+  const auto workingFolder = findCmdLineKey(argc, argv, "-cd");
```

Comment:
Really only use `auto` if the type is complicated (e.g. some templated construct of sorts), otherwise
Avoid `auto` if writing out the type is not a burden. In _particular_ as the reader needs to know the type to understand that it is compatible with the `chdir()` function below.

So make it `const char* workingFolder = ...`

Reply:
no opinion, I wish clang-format-14 would just do something consistent.

Reply:
fixed